assemblyline_core/ingester/ingester.py

```diff
@@ -674,19 +673,19 @@ def stamp_filescore_key(task: IngestTask, sha256=None):
         if not sha256:
             sha256 = task.submission.files[0].sha256
 
-        key = task.scan_key
+        key = task.submission.scan_key
 
         if not key:
             key = task.params.create_filescore_key(sha256)
-            task.scan_key = key
+            task.submission.scan_key = key
 
         return key
 
     def completed(self, sub):
         """Invoked when notified that a submission has completed."""
         # There is only one file in the submissions we have made
         sha256 = sub.files[0].sha256
-        scan_key = sub.params.create_filescore_key(sha256)
+        scan_key = sub.scan_key or sub.params.create_filescore_key(sha256)
```

Comment:
Why the OR? Shouldn't it alwas be there?

Reply:
There should be a ticket to remove this in a few months so we don't forget it then.